internal/scripts/build.js

```diff
@@ -26,9 +26,18 @@ Object.keys(config('bundles'))
     const compiler = webpack(webpackConfigFactory({ target: bundleName, optimize }));
     compiler.run((err, stats) => {
       if (err) {
-        console.error(err);
+        console.error(err.stack || err);
+        if (err.details) {
+          console.error(err.details);
+        }
+        process.exitCode = process.exitCode || 1;
```

Comment:
Sorry for my ignorance but could you please explain the reasoning for setting the process exit code?